Tools/doctools.patch

```diff
@@ -0,0 +1,43 @@
+diff --git a/Editor/Sources/Services/Doc/Documentation.cs b/Editor/Sources/Services/Doc/Documentation.cs
```

Comment:
Wondering if this mechanism I put in place will turn out to be too brittle. Just thinking of them changing something upstream that breaks the patch and then it will hit a random PR in this repo. Is embedding a copy of the package the better solution after all? What do you think?

Reply:
My plan is to next go and land this change into upstream. Doing that will not make our repo break (as patch will be a no-op), but allows us to land this first.

Reply:
I don't expect doctools to break much if at all. If this situation arises we can always pin git checkout to a specific hash. But I would prefer to always be up-to-date until it fails.